x-pack/plugins/apm/common/environment_filter_values.ts

```diff
@@ -55,6 +55,18 @@ export function getEnvironmentEsField(environment: string) {
   return { [SERVICE_ENVIRONMENT]: environment };
 }
 
+export function getEnvironmentKuery(environment: string) {
```

Comment:
Can't you use this function instead?

 https://github.com/elastic/kibana/blob/main/x-pack/plugins/apm/common/environment_filter_values.ts#L46-L56